indra/tests/test_pathfinding.py

```diff
@@ -181,6 +181,23 @@ def test_bfs():
         error = True
     assert error
 
+    # Test edge filter
+    # With belief cutoff at 0.75, we should remove:
+    # ('A2', 'B1'), ('A3', 'B2'), ('A4', 'B2'), ('B2', 'C1'), ('B3', 'C1')
+    def _filter_func(g, u, v):
+        return g.edges[(u, v)]['belief'] > 0.75
```

Comment:
```suggestion
        return g.edges[u, v]['belief'] > 0.75
```

Reply:
Wait, I always use `G.edges[(u, v)]` and I see now the documentation does it your way: https://networkx.org/documentation/stable/reference/introduction.html#graph-reporting

I'm guessing they provide both ways since an edge is a tuple, e.g. `[e for e in g.edges]`

Reply:
Yes, it will also work if you do:
```python
e = (u,v)
g.edges[e]
```

This is not a specific feature of networkx, but more generally how the square bracket syntax functions.